shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/java/org/apache/shardingsphere/sql/parser/sqlserver/visitor/statement/impl/SQLServerDCLStatementSQLVisitor.java

```diff
@@ -353,4 +355,10 @@ public ASTNode visitSetUser(final SetUserContext ctx) {
         }
         return result;
     }
+    
+    @Override
+    public ASTNode visitRevert(final RevertContext ctx) {
+        SQLServerRevertStatement result = new SQLServerRevertStatement();
+        return result;
```

Comment:
@ThanoshanMV `return new SQLServerRevertStatement()` is better.

Reply:
Thank you @strongduanmu. I've made changes.